src/NuGet.Core/NuGet.ProjectModel/PackageSpecReferenceDependencyProvider.cs

```diff
@@ -326,6 +326,19 @@ private List<LibraryDependency> GetDependenciesFromSpecRestoreMetadata(PackageSp
                 var targetFrameworkInfo = packageSpec.GetTargetFramework(targetFramework);
                 dependencies.AddRange(targetFrameworkInfo.Dependencies);
 
+                if (packageSpec.RestoreMetadata?.CentralPackageVersionsEnabled ?? false)
```

Comment:
```suggestion
                if (packageSpec.RestoreMetadata?.CentralPackageVersionsEnabled == true)
```

Positive conditions read a bit better.

Reply:
Updated.